starknet_devnet/blocks.py

```diff
@@ -120,15 +126,16 @@ async def generate(
                 tx_signatures=[signature],
                 event_hashes=[],
                 sequencer_address=state.general_config.sequencer_address
-            )
-
-        transaction_receipts = (transaction.get_execution(),)
+                )
```

Comment:
here indentation for the whole calculate_block_hash?

Reply:
yes that is correct!

Reply:
The indentation might be syntactically correct, but the readability could be improved by adding an extra tab to the `calculate_block_hash` arguments. However, this might be irrelevant if the comment on the line 113 is resolved.